.gitpod.yml

```diff
@@ -0,0 +1,32 @@
+image:
+  file: .gitpod.dockerfile
+
+github:
+  prebuilds:
+    branches: true
+
+ports:
+- port: 3000
+  onOpen: ignore
+# We open this port because it's the port that nginx is serving and which is federating the other services (web app and chain in this case)
+- port: 8000
+  onOpen: open-preview
+- port: 8900
```

Comment:
Are all these ports necessary?

Reply:
Obviously 3000 for the app, 8000 for nginx, and 8888 for nodeos not familiar with the rest

Reply:
Yes, any port not listed here (that is actually being used) will spam the user with a little pop-up box, so I added 8900 (keosd) to stop that.  All these ports are used by the system.